CUE4Parse/UE4/Assets/Exports/StaticMesh/FStaticMeshRayTracingProxyLOD.cs

```diff
@@ -32,18 +32,27 @@ public FStaticMeshRayTracingProxyLOD(FAssetArchive Ar)
 
         bOwnsRayTracingGeometry = Ar.ReadBoolean();
 
+        if (Ar.Game >= EGame.GAME_UE5_6)
+            bBuffersInlined = Ar.ReadBoolean(); // always false ???
 
-        if (bBuffersInlined) // always false ???
+        if (bBuffersInlined)
```

Comment:
ive looked at the changes yesterday myself and i think you need mode changes here or am i missing something? the ue code in the true case seems to read a third function `SerializeRayTracingGeometry` inbetween `SerializeBuffers` and `SerializeMetaData` and in the else part it should always read `StreamableData` and then afterwards read `SerializeMetadata` when ue 5.6.

Ive used [this](https://github.com/epicgames/unrealengine/blob/3b280e7c0b95076e21deaa92c5f4e0222d5775d9/Engine/Source/Runtime/Engine/Private/StaticMesh.cpp#L1763) as the reference

Reply:
That makes more sense and follows the order of serialization. The [additional int]() turns was **baseIndex** in **FByteBulkData**. I was using this as my [reference](https://github.com/EpicGames/UnrealEngine/blob/295133e998c844c02461248fbe07d9cae8820b9d/Engine/Source/Runtime/Engine/Private/StaticMesh.cpp#L1727)

Reply:
To further add to this https://github.com/FabianFG/CUE4Parse/blob/5f6770f0e6f4c6a3b1534a068e742ed77f27fd16/CUE4Parse/UE4/Assets/Exports/StaticMesh/FStaticMeshRayTracingProxyLOD.cs#L61